R/overrideGraphics.R

```diff
@@ -14,11 +14,16 @@ overrideGraphics <- function(name, object, ...) {
         params <- append(list(object), params)
     }
     
-    
-    defFun <- getFromNamespace(ifelse(existsFunction(paste0(name,".default")), paste0(name,".default"), name), 'graphics')
+    base.package <- ifelse(length(grep(pattern="error_bar", x=name)) > 0, "gsplot", "graphics")
+    defFun <- getFromNamespace(ifelse(existsFunction(paste0(name,".default")), paste0(name,".default"), name), base.package)
     
     names(params)[which(names(params) == "")] <- names(formals(defFun))[which(names(params) == "")]
     
-    do.call(getFromNamespace(name, "graphics"), params)
+    if(base.package == "gsplot"){
```

Comment:
There was a recursive problem. It was pretty funky actually. So this is allowing us to make new functions (like error_bars), set a default and gsplot method.